compiler/fir/checkers/src/org/jetbrains/kotlin/fir/analysis/checkers/declaration/FirSealedSupertypeChecker.kt

```diff
@@ -29,19 +31,29 @@ object FirSealedSupertypeChecker : FirClassChecker() {
     }
 
     private fun checkGlobalDeclaration(declaration: FirClass, context: CheckerContext, reporter: DiagnosticReporter) {
-        for (it in declaration.superTypeRefs) {
-            val classId = it.coneType.classId ?: continue
+        val subclassPackage = declaration.classId.packageFqName
+        for (superTypeRef in declaration.superTypeRefs) {
+            val classId = superTypeRef.coneType.classId ?: continue
```

Comment:
Super minor: if `classSymbol` below is renamed to `superClass`, isn't it aligned if we renamed this to `superClassId`? :)